MOBILE_API.xml

```diff
@@ -8236,12 +8236,15 @@
                 This parameter is filled for supporting OEM proprietary data exchanges.
             </description>
         </param>
-        <param name="url" type="String" maxlength="1000" mandatory="false">
+        <param name="url" type="String" mandatory="false" since="6.1">
             <description>
                 Optional URL for HTTP requests.
                 If blank, the binary data shall be forwarded to the app.
                 If not blank, the binary data shall be forwarded to the url with a provided timeout in seconds.
             </description>
+            <history>
+                <param name="url" type="String" maxlength="1000" mandatory="false" since="3.0" until="6.1"/>
```

Comment:
```suggestion
                <param name="url" type="String" maxlength="1000" mandatory="false" since="3.0" until="7.0"/>
```
RPC spec version has been bumped to 7.0

Reply:
@ShobhitAd 
Fixed in:https://github.com/smartdevicelink/rpc_spec/pull/225/commits/d2bb2bc092224bc395eda1da78bad4a29434bd99